practice-areas/engineering/accessibility.md

```diff
@@ -4,22 +4,22 @@ title: Accessibility
 
 # Accessibility
 
-We implement Section 508 standards and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
+We implement Section 508 compliant sites and strive to meet the latest Web Content Accessibility Guidelines (WCAG). We do this so that everyone can access our sites, even if they have a permanent, temporary or situtational disability. The Engineering team is very aware of the advantages of open standards, and semantic code for building future-compatible applications. CivicActions has an [Accessibility Practice Area](/en/latest/practice-areas/accessibility/), but is also a component of all other practice areas at CivicActions.
```

Comment:
@mgifford can you change the path to 'Accessibility Practice Area' to `../accessibility/README.md`.

Reply:
Yup.